### ISSUE TITLE
Installing with Ember-CLI 2.2.0.beta.3 doesn't install bower dependencies 

### ISSUE THREAD

**Issue by visoft** [danielspaniel/ember-data-factory-guy#170]
I'm running Ember-CLI 2.2.0.beta.3 and I'm getting the following error when trying to test:

```
ENOENT, no such file or directory .../tmp/source_map_concat-input_base_path-fye3Iewj.tmp/0/bower_components/jquery-mockjax/dist/jquery.mockjax.js'
```

Turns out, there is no jquery-mockjax directory in my `bower_components` directory. I'm sure I can install it manually, but is this an Ember-CLI bug or a problem with ember-data-factory-guy? I cleanly installed ember-data-factory-guy using v2.1.3.

**Comment by danielspaniel** [comment]
Not sure @visoft, did you try installing in new project? seems like it. I will try it myself and see if I can replicate on cli 2.2.0.beta.3

**Comment by visoft** [comment]
I upgraded a project from cli 1.13.7, it's [this project](https://github.com/visoft/semaphore_bookcase_client). Master is at a point where I just upgraded and have yet to install ember-data-factory-guy.

**Comment by danielspaniel** [comment]
sorry to say, I can not recreate this. I did not use your repo, but I did make a new ember project with ember-cli 1.13.3 then I updated to cli 2.2.0-beta.3 .. then I ran ember init, and overrode all files, like bower.json and package.json ( not usually a great idea to override those two ) .. then ember install ember-data-factory-guy  install mockjax ok.  You must have done a particular combination of something that I am not sure how to recreate.
